src/test/FreeC/Backend/Agda/Converter/FuncDeclTests.hs

```diff
@@ -0,0 +1,90 @@
+module FreeC.Backend.Agda.Converter.FuncDeclTests
+  ( testConvertFuncDecls
+  )
+where
+
+import           Test.Hspec
+
+import           FreeC.Backend.Agda.Converter.FuncDecl
+                                                ( convertFuncDecls )
+import           FreeC.Backend.Agda.Pretty      ( )
+import           FreeC.IR.DependencyGraph
+import           FreeC.Monad.Class.Testable
+import           FreeC.Monad.Converter
+import           FreeC.Test.Parser
+import           FreeC.Test.Environment
+import           FreeC.Test.Expectations
+import           FreeC.Pretty                   ( showPretty )
+
+-- | Parses the given IR function declaration, converts it to Agda using
+--   'convertFuncDecls' and sets the expectation that the resulting
+--   Agda code equals the given expected output modulo white space.
```

Comment:
As far as I remember consensus in the discussion on Slack was to spell whitespace without whitespace :D

```suggestion
--   Agda code equals the given expected output modulo whitespace.
```